src/observers/strategies/cpu_contention.hpp

```diff
@@ -50,26 +50,22 @@ class CpuContentionStrategy : public RevocationStrategy {
       const SerenityConfig& _config,
       const lambda::function<usage::GetterFunction>& _cpuUsageGetFunction)
       : RevocationStrategy(Tag(QOS_CONTROLLER, "CpuContentionStrategy")),
-        cpuUsageGetFunction(_cpuUsageGetFunction),
-        cooldownCounter(None()),
-        estimatorDisabled(false) {
+        getCpuUsage(_cpuUsageGetFunction) {
     SerenityConfig config = CpuContentionStrategyConfig(_config);
-    this->cfgCooldownTime = config.getU64(strategy::CONTENTION_COOLDOWN);
-    this->cfgDefaultSeverity = config.getD(strategy::DEFAULT_CPU_SEVERITY);
   }
 
-  RevocationStrategyFunction decide;
+  Try<QoSCorrections> decide(ExecutorAgeFilter* ageFilter,
```

Comment:
IMOiIt is worse solution. `RevocationStrategyFunction` looks cleaner. However, leave it if you think it's better, i don't want to stop the push because of that, not important issue.